code/modules/photography/_pictures.dm

```diff
@@ -68,8 +68,11 @@
 	.["pixel_size_y"] = psize_y
 	.["blueprints"] = has_blueprints
 	.["logpath"] = logpath
+	options[SCHEMA_VERSION] = "1.0"
 
 /datum/picture/deserialize_list(list/input, list/options)
+	if(options[SCHEMA_VERSION] != "1.0")
```

Comment:
This serialize_list and deserialize_list stuff was added by JSON logging, right? We're not about to make it so nobodys persisted photo albums can't load?

Reply:
You didn't answer this. What is going to happen to people's pictures?